### PR TITLE
Tooltip footer, relocation of browsermanager init, remove version from footer status bar

### LINKED ISSUE #278: Relocation of initComponents()
The creation of browserManager instance need to be after the stage is created. As JavaFX webview needs it. 

Currently it is created before the stage is created.

Any suggestions guys, on where to move to?

```

    @Override
    public void init() throws Exception {
        logger.info("Initializing app ...");
        super.init();
        initConfig();
        Config.setConfig(Config.getConfig());
        initPrefs();
        BrowserManager.initializeBrowser();
        //TODO: should this be here? looks out of place
        initComponents();
    }
```

### PR DESCRIPTION
**Pull request by yl-coder**
Fixes #278

### CODE REVIEW

**Comment by m133225** [comment]
Can't it be called in `BrowserManager`'s constructor? Didn't realise you were also changing this, will remove in my PR.

**Comment by yl-coder** [comment]
@m133225 The initialise method of browser in mainApp must be called in a non ui. So maybe thats the reason why it is placed there instead at the mainApp.

File: src/main/java/address/controller/StatusBarFooterController.java [hunk 1]
```diff
@@ -55,8 +56,12 @@ private void syncSchedulerOnTimeOut() {
...
         this.syncStatusBar = new StatusBar();
         this.updaterStatusBar = new StatusBar();
-        this.updaterStatusBar.getLeftItems().add(saveLocText);
-
+        saveLocText.setTextAlignment(TextAlignment.LEFT);
+        Tooltip tp = new Tooltip();
+        tp.textProperty().bind(saveLocText.textProperty());
+        saveLocText.setTooltip(tp);
+        this.updaterStatusBar.getRightItems().add(saveLocText);
+        saveLocText.setVisible(false);
```
Comment (HansNewbie) [inline src/main/java/address/controller/StatusBarFooterController.java]: Abstract this to a method? Setting up of tooltip here breaks the abstraction level.

**Comment by HansNewbie** [comment]
No more comments.

**Comment by damithc** [comment]
Why did we remove version from status bar?

**Comment by yl-coder** [comment]
Since we have the save location detail in the footer, i guess we can move the version to the about section? Since the user does not need to know about the version all the time.

**Comment by damithc** [comment]
I see. Yes, that's the better place for version. However, we should add some more stuff to the footer to simulate a legit footer showing many different data items.

**Comment by yl-coder** [comment]
Right. Will try to introduce more stuff into the footer.

**Pull request merged**